kie-server-parent/kie-server-services/kie-server-services-jbpm-cluster/src/main/java/org/kie/server/services/jbpm/cluster/EJBCacheInitializer.java

```diff
@@ -0,0 +1,43 @@
+package org.kie.server.services.jbpm.cluster;
+
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+
+import org.infinispan.Cache;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.jgroups.Address;
+import org.kie.api.cluster.ClusterAwareService;
+import org.kie.api.cluster.ClusterNode;
+
+@Singleton
+@Startup
+public class EJBCacheInitializer {
+
+    public static final String CACHE_NAME_LOOKUP = "java:jboss/infinispan/container/jbpm";
+    
+    public static final String CACHE_NODES_NAME_LOOKUP = "java:jboss/infinispan/cache/jbpm/nodes";
+    
+    public static final String CACHE_JOBS_NAME_LOOKUP = "java:jboss/infinispan/cache/jbpm/jobs";
+    
+    // this enforce the cache initializer
+
+    @Resource(lookup = CACHE_NAME_LOOKUP)
+    private EmbeddedCacheManager cacheManager;
+
+    @Resource(lookup = CACHE_NODES_NAME_LOOKUP)
+    private Cache<Address, ClusterNode> nodesCache;
+
+    @Resource(lookup = CACHE_JOBS_NAME_LOOKUP)
+    private Cache<String, List<Long>> jobsCache;
```

Comment:
this is a long story. infinispan cache are started on demand in wildfly so the only way to start them (because EAGER initilization in EAP is deprecated is referencing them in the ejb.